### PR TITLE
Fetch dataset from Spine Generic

### LINKED ISSUE #26: Indicate user to fetch dataset from spine generic
"csa-atrophy" dataset is no longer hosted on "openneuro" but on "spine generic project", https://github.com/spine-generic.

TODO:
- update README
- update csa_fetch_dataset

### PR DESCRIPTION
**Pull request by PaulBautin**
"csa-atrophy" test dataset is no longer hosted on "openneuro" but on "spine generic project", https://github.com/spine-generic/data-multi-subject. An update is required to reflect this change.

Done: 
- updated README
- remove csa_fetch_dataset
- use data-multi-subject last release https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801

FIX #26

### CODE REVIEW

**Comment by PaulBautin** [comment]
Ok, are we settling on https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801 for csa-atrophy? @jcohenadad

File: README.md [hunk 1]
```diff
@@ -30,18 +30,15 @@ Download (or git clone) this repository:
...
-  - To fetch/sync dataset run command: (this file should be edited according to your needs)
+Fetch dataset:
+Suggested testing dataset must be downloaded from "Spine Generic Public Database". To download latest version of the whole multi-subject dataset run commands:
   ~~~
-  ./csa_fetch_dataset.sh
-  ~~~
-  - To fetch specific version of openneuro repository (version: 1.0.5, Files: 2501, Size: 7.9GB, Subjects: 248) follow instructions to set openeuro CLI: https://www.npmjs.com/package/openneuro-cli
-  ~~~
-  openneuro download --snapshot 1.0.5 ds001919 data
+  curl -o spinegeneric.zip -L https://github.com/spine-generic/data-multi-subject/archive/master.zip
```
Comment (jcohenadad) [inline README.md:36]: should point to specific release

**Comment by jcohenadad** [comment]
> Ok, are we settling on https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801 for csa-atrophy? @jcohenadad

for now yes, when the release is updated, it's easy to update the URL here

**Comment by PaulBautin** [comment]
I think PR can be closed @jcohenadad . Most important features:
 - update README so that users download dataset from spine generic  https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801
- remove fetch_dataset script

File: README.md [hunk 2]
```diff
@@ -19,29 +19,27 @@ Dataset/
...
+Suggested testing dataset must be downloaded from "Spine Generic Public Database". To download latest version of the whole multi-subject dataset run commands:
   ~~~
-  openneuro download --snapshot 1.0.5 ds001919 data
+  curl -o spinegeneric.zip -L https://github.com/spine-generic/data-multi-subject/archive/r20200801.zip
+  unzip spinegeneric.zip
   ~~~
 Run the following script within the Dataset folder to extract CSA. This script can be run on desired subjects using flag -include and in parallel processing using flag -jobs.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results process_data.sh
+sct_run_batch -path-data data-multi-subject-master -path-output csa_atrophy_results process_data.sh
```
Comment (jcohenadad) [inline README.md:42]: are you sure this is the correct folder name now that you've changed the url to "https://github.com/spine-generic/data-multi-subject/archive/r20200801.zip"?

File: README.md [hunk 3]
```diff
@@ -57,5 +55,5 @@ After running the analysis, check your Quality Control (QC) report by opening th
 The bash script outputs all effectuated manual labelings to 'results_corrected/seg_manual'.
 It is now possible to re-run the whole process, pointing to the manual corrections. With the command below labeling will use the manual corrections present in 'seg_manual', otherwise labeling will be done automatically.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
+sct_run_batch -path-data data-multi-subject-master -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
```
Comment (jcohenadad) [inline README.md:58]: same as https://github.com/sct-pipeline/csa-atrophy/pull/28/files#r464656202

**Comment by jcohenadad** [comment]
@PaulBautin the PR body seems unfinished and contains an unresolved "TODO". Please cleanup before we merge

File: README.md (hunk 2 above)
Comment (PaulBautin) [reply README.md]: `sct_run_batch` now uses a config file that is not present in this PR (i have implemented this comment as a reminder in commit https://github.com/sct-pipeline/csa-atrophy/pull/28/commits/57b8c1d7ebdb74573131a4634fead5aa3f4243a4).

File: README.md (hunk 3 above)
Comment (PaulBautin) [reply README.md]: same as > `sct_run_batch` now uses a config file that is not present in this PR (i have implemented this comment as a reminder in commit [57b8c1d](https://github.com/sct-pipeline/csa-atrophy/commit/57b8c1d7ebdb74573131a4634fead5aa3f4243a4)).

File: README.md [hunk 4]
```diff
@@ -19,29 +19,29 @@ Dataset/
...
-  openneuro download --snapshot 1.0.5 ds001919 data
+  curl -o spinegeneric_r20200801.zip -L https://github.com/spine-generic/data-multi-subject/archive/r20200801.zip
+  unzip spinegeneric_r20200801.zip
   ~~~
+To run csa-atrophy pipeline on a sub-dataset of spine-generic there are 2 options: either create a new folder with a sub-dataset (make sure to update config file path_data) or use flag -include in sct_run_batch to select one subject at a time.
+
 Run the following script within the Dataset folder to extract CSA. This script can be run on desired subjects using flag -include and in parallel processing using flag -jobs.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results process_data.sh
+sct_run_batch -path-data spinegeneric_r20200801 -path-output csa_atrophy_results process_data.sh
```
Comment (jcohenadad) [inline README.md:44]: sct_run_batch now uses -task flag

Comment (jcohenadad) [reply README.md:44]: same thing below

Comment (PaulBautin) [reply README.md:44]: I just did a pull from SCT repo, i don't see flag -task in sct_run_batch. However, i'm thinking of using flag -config so no need for other flags.

**Comment by PaulBautin** [comment]
I have corrected path to dataset in the README and added a comment on users options to test the spine-generic dataset without processing all subjects.

File: README.md (hunk 4 above)
Comment (jcohenadad) [reply README.md:44]: it was actually changed to -script: 
https://github.com/neuropoly/spinalcordtoolbox/blob/master/scripts/sct_run_batch.py#L143

it is always important to test the code that you write

File: README.md [hunk 5]
```diff
@@ -57,5 +57,5 @@ After running the analysis, check your Quality Control (QC) report by opening th
 The bash script outputs all effectuated manual labelings to 'results_corrected/seg_manual'.
 It is now possible to re-run the whole process, pointing to the manual corrections. With the command below labeling will use the manual corrections present in 'seg_manual', otherwise labeling will be done automatically.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
+sct_run_batch -path-data spinegeneric_r20200801 -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
```
Comment (jcohenadad) [inline README.md:60]: same here-- that will crash-- please test your procedure, then let me know when your code is ready for a review

**Comment by PaulBautin** [comment]
@jcohenadad, i will close this PR without merging it, if it is ok with you? The scope is small and the branch is behind master, i am not able to test procedure easily. I have already merged this PR in #31.

**Pull request closed without merging**